templates/nutrient_levels.tt.html

```diff
@@ -0,0 +1,28 @@
+<!-- Nutrient levels template -->
+[% IF (nutrition_grade == "exists") && (nutrient_levels != '') %]
+    [% IF nutrient_levels != '' %]
+        <div class="small-12 xlarge-6 columns">
+            <h4>[% lang('nutrient_levels_info') %]
+                <a href="[% lang('nutrient_levels_link') %]" title="[% lang('nutrient_levels_info') %]">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" class="icon" aria-hidden="true" focusable="false"><path d="M0 0h24v24H0z" fill="none"/><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"/></svg>
```

Comment:
I don't think it's a good idea to put the SVG in there, we should pass a parameter to the template with the output of display_icon()

Reply:
Or maybe just call display_icon() directly in the template